staging/src/k8s.io/apiserver/plugin/pkg/audit/webhook/webhook.go

```diff
@@ -58,9 +60,13 @@ const (
 	//
 	// TODO(ericchiang): Make these value configurable. Maybe through a
 	// kubeconfig extension?
-	defaultBatchBufferSize = 1000        // Buffer up to 1000 events before blocking.
-	defaultBatchMaxSize    = 100         // Only send 100 events at a time.
-	defaultBatchMaxWait    = time.Minute // Send events at least once a minute.
+	defaultBatchBufferSize = 10000            // Buffer up to 10000 events before starting discarding.
+	defaultBatchMaxSize    = 400              // Only send up to 400 events at a time.
+	defaultBatchMaxWait    = time.Minute      // Send events at least once a minute.
```

Comment:
Do we have any stats on how often this happens? It might be worth adding a prometheus metric for it... (I.e. maybe it can be shorter without increasing QPS much)

Reply:
Almost never, in 3-10 nodes cluster there're 10 events generated per second even nothing is happening. I will lower this to 30, it's closer to the current amount of time needed to collect a batch in a small cluster

Reply:
Thanks for reminding!

Reply:
Done